### PR TITLE
add NEWS.md entry re. raising one or negative one to a negative integer power

### PR DESCRIPTION
**Pull request by Sacha0**
Clears #18342 from https://github.com/JuliaLang/julia/issues/21475#issuecomment-301185844. Best!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -337,6 +337,9 @@ Library improvements
   * A new `Dates.Time` type was added that supports representing the time of day
     with up to nanosecond resolution ([#12274]).
 
+  * Raising one or negative one to a negative integer power no longer
+    throws a `DomainError` ([#18342]).
```
Comment (StefanKarpinski) [inline NEWS.md]: This seems like it could use a little more elaboration – i.e. what does it now do instead of throwing an error?

Comment (Sacha0) [inline NEWS.md]: Better? :)

File: NEWS.md [hunk 2]
```diff
@@ -337,6 +337,12 @@ Library improvements
   * A new `Dates.Time` type was added that supports representing the time of day
     with up to nanosecond resolution ([#12274]).
 
+  * Raising one or negative one to a negative integer power formerly threw a `DomainError`.
+    One raised to any negative integer power now yields one, negative one raised to any
+    negative even integer power now yields one, and negative one raise to any negative
```
Comment (tkelman) [inline NEWS.md]: raised to any

Comment (Sacha0) [inline NEWS.md]: Fixed on push. Thanks!

**Review by StefanKarpinski — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (StefanKarpinski) [inline NEWS.md]: Beautiful!

**Pull request merged**